x-pack/plugins/security_solution/cypress/tasks/alerts_grouping.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  GROUP_COUNT,
+  GROUP_LEVEL_SELECTOR,
+  GROUP_OPTION_SELECTOR,
+  GROUP_PAGINATION_SELECTOR,
+  GROUP_SELECTOR,
+  GROUP_SELECTOR_OPTION,
+} from '../screens/alerts';
+
+export const selectGroup = (groupName: string) => {
+  cy.get(GROUP_SELECTOR).click();
+  cy.get(GROUP_OPTION_SELECTOR(groupName)).click();
+};
+
+// TODO: @Glo please help here, these clicks are flakey
```

Comment:
@MadameSheema I need help getting this click to be less flaky, not sure what is going on